src/js/components/Button/Button.js

```diff
@@ -86,9 +86,10 @@ class Button extends Component {
 
     let buttonIcon = icon;
     // only change color if user did not specify the color themselves...
-    if (primary && icon && !icon.props.color) {
+    if (icon && !icon.props.color) {
```

Comment:
This appears to break things: https://deploy-preview-2891--sad-tereshkova-173d07.netlify.com/?selectedKind=Button&selectedStory=Icon%20Label&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Fstories%2Fstories-panel